tests/scClient.test.ts

```diff
@@ -100,8 +100,10 @@ describe("ScClient", () => {
         expect.objectContaining({
           type: "create_elements_by_scs",
           payload: expect.arrayContaining([
-            "my_class -> node1;;",
-            "my_class -> rrel_1: node1;;",
+            {scs: "my_class -> node1;;",
+             outputStructure: 0},
```

Comment:
Can we don't point outputStructure value? Don't change this test. Tests static state shows backward compatibility